### ISSUE TITLE
Feature: package that defines a function/method

### ISSUE THREAD

**Issue by yko** [plicease/Class-Inspector#1]
`Class::Inspector` provides an information on which namespace a function/method is installed to which is useful. Thanks!

It could be extended to cover cases when a subroutine was imported, installed or generated by another package. This would cover cases different from inheritance through ISA array, but direct installation into a namespace: Exporter-alike functionality and accessors installed from the outside. 

At some point I had [basic implementation](https://github.com/yko/devel-sublist/blob/master/lib/Devel/Sublist.pm#L128) of this feature, but never published it on CPAN.
Would such feature fit `Class::Inspector` purpose?

**Comment by plicease** [comment]
Thanks to Adam Kennedy who wrote the code :)

My goal is to keep 5.6-5.26+ compatibility without introducing any new dependencies.  If we can provide this functionality within those constraints, yeah I think it would be useful.

**Comment by yko** [comment]
If you don't mind usage of `B`, I'll give it a try.

Out of curiosity, what is the rationale to keep support for 5.6?

**Comment by plicease** [comment]
B is core since 5.005, so sure.

This is an older module with a number of reverse dependencies, at least some of which still support 5.6 I suspect, and I don't want to break that without a compelling reason.

**Comment by yko** [comment]
That sounds healthy. Thanks for explaining!

**Comment by mrenvoize** [comment]
Wondering if this ever went anywhere.. I just got stun by it not fully understanding how @ISA inhertiance compares to direct imports.

**Comment by plicease** [comment]
I've not been working on this, but patches welcome!

**Comment by yko** [comment]
Indeed, it seems I got distracted. I doubt I could dedicate any time to this in the observable future.
Just in case there's someone willing to take on this patch, I'd be happy to act as a supporter, etc.
